src/sqlfluff/rules/L005.py

```diff
@@ -43,20 +43,22 @@ class Rule_L005(BaseRule):
     groups = ("all", "core")
     crawl_behaviour = SegmentSeekerCrawler({"comma"})
 
-    def _eval(self, context: RuleContext) -> Optional[LintResult]:
+    def _eval(self, context: RuleContext) -> List[LintResult]:
```

Comment:
Nit: `BaseRule._eval()` returns `EvalResultType`. The rule implementations (like this one) often return something else. Would be cool at some point if the rules were consistent.